Cargo.toml

```diff
@@ -20,6 +20,12 @@ codegen-units = 1
 panic = "abort"
 strip = "symbols"
 
+[profile.release.build-override]
+inherits = "dev.build-override"
+
+[profile.release.package."*"]
+opt-level = 3
```

Comment:
wait why don't we just use 3 for the entire profile, instead of just the deps? we've got a lot of code ourselves too

Reply:
> wait why don't we just use 3 for the entire profile, instead of just the deps? we've got a lot of code ourselves too

I thought that stuff in `binstalk` probably isn't the bottleneck, but anyway now it switched to using `-O3` for everything in release build.